rest-api/dao/genomics_dao.py

```diff
@@ -38,3 +114,33 @@ def upsert(session):
         written += 1
       return written
     return self._database.autoretry(upsert)
+
+  def bulk_update_validation_status(self, member_id_status_pair_iterable):
+    with self.session() as session:
+      self.bulk_update_validation_status_with_session(member_id_status_pair_iterable, session)
+
+  def bulk_update_validation_status_with_session(self, session, member_id_status_pair_iterable):
+    now = clock.CLOCK.now()
+    status_case = sqlalchemy.case(
+      {int(GenomicValidationStatus.VALID): now},
+      value=sqlalchemy.bindparam('status'),
+      else_=None
+    )
+    query = (
+      sqlalchemy
+        .update(GenomicSetMember)
+        .where(GenomicSetMember.id == sqlalchemy.bindparam('member_id'))
+        .values({
+          GenomicSetMember.validationStatus.name: sqlalchemy.bindparam('status'),
+          GenomicSetMember.validatedTime.name: status_case
+        })
+    )
+    parameter_sets = [
+      {
+        'member_id': member_id,
+        'status': int(status),
+        'time': now if status == GenomicValidationStatus.VALID else None,
+      }
+      for member_id, status in member_id_status_pair_iterable
+    ]
+    session.execute(query, parameter_sets)
```

Comment:
This may need to be re-worked to update the `version` column as well but I'm not sure I can make this error-out if the version number is not the expected version.
@robabram thoughts?

Reply:
After talking with @robabram about the `version` field. We arrived at not needing it for this model which means we can safely have this bulk update.